flink-core/src/main/java/org/apache/flink/api/common/io/SinkUtils.java

```diff
@@ -18,16 +18,13 @@
 
 package org.apache.flink.api.common.io;
 
-import org.apache.flink.annotation.Internal;
-
 import java.io.Serializable;
 import java.time.Duration;
 import java.util.concurrent.Semaphore;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
 /** Utility class for sinks. */
-@Internal
 public class SinkUtils implements Serializable {
```

Comment:
Let's mark this as experimental, in line with the sink.
(everything no annotated is implicitly internal ;))